### DISCUSSION TITLE
64 bit Buster support

### DISCUSSION THREAD

**Discussion by wachux** [aquaticus/nexus433#42] [Q&A]
HI, 
I have a hard time installing on 64 bit Buster. Is it supported ?

**Comment by aquaticus** [comment]
Precompiled packages supports only 32 bits. 
To use on 64bit you must compiled it locally.

**Reply by wachux** [reply]
thanks for quick reply - are you able to help me doing this ?

**Comment by aquaticus** [comment]
Everything is described in README. Start from https://github.com/aquaticus/nexus433#manual-build-and-installation

**Comment by wachux** [comment]
I did it couple of times before with no luck. I get this error:
Failed to start nexus433.service: Unit nexus433.service has a bad unit file setting.

**Comment by wachux** [comment]
when i run: service nexus433 status, i get
 nexus433.service - 433MHz sensor receiver
     Loaded: bad-setting (Reason: Unit nexus433.service has a bad unit file setting.)
     Active: inactive (dead)

Oct 20 15:59:40 raspberrypi systemd[1]: nexus433.service: Service has no ExecStart=, ExecStop=, or SuccessAction=. Refusing.
Oct 20 16:00:06 raspberrypi systemd[1]: nexus433.service: Service has no ExecStart=, ExecStop=, or SuccessAction=. Refusing.

**Reply by aquaticus** [reply]
can you paste here content of `nexus433.service`? It should be located in `/usr/local/lib/systemd/system/` (I assume you executed `make install`)

**Comment by wachux** [comment]
[Unit]
Description=433MHz sensor receiver
After=network-online.target

[Service]
Type=simple
User=root
ExecStart=

[Install]
WantedBy=multi-user.target

**Comment by aquaticus** [comment]
There's a bug in cmake (I probably introduced it in one of the latest commits) which results in invalid `nexus433.service`.
You have to modify the line with `ExecStart=`; add path to executable
`ExecStart=/usr/local/bin/nexus433`

**Comment by wachux** [comment]
It works now :) Thanks

**Comment by aquaticus** [comment]
Thanks for spotting this issue.
It's already fixed and make should produce valid installation files.
